test/specs/cartesian/YAxisSpec.js

```diff
@@ -194,4 +189,40 @@ describe('<YAxis />', () => {
     expect(wrapper.find('svg').children.length).to.equal(1);
     expect(wrapper.find('svg noscript').children.length).to.equal(1);
   });
+
+  it('Render identical ticks when data is hidden and includeHidden is true', () => {
```

Comment:
Would it make sense to add a separate test for the xAxis?

Reply:
I did have a look at the XAxis tests, and made the decision to not include them, as I didn't see much crossover in testing responsibilities  between the two files, and the code path should be covered by the YAxis test, since the logic is shared. If you think its the right move to add a test there, then I certainly can.